kratos/tests/test_model_part.py

```diff
@@ -228,7 +228,9 @@ def test_model_part_properties(self):
         self.assertEqual(model_part.NumberOfProperties(), 0)
         self.assertEqual(model_part.NumberOfProperties(0), 0)
 
+        self.assertEqual(model_part.HasProperties(1), False)

```

Comment:
@philbucher this is a test

Reply:
I mean you should also check that this property does NOT exist in a submodelpart
-> what Riccardo was explaining